docs/_data/toc.yml

```diff
@@ -65,9 +65,7 @@
   links:
     - title: Architecture ↆ
       url: concepts/architecture
-      children:
-        - title: Adapters
-          url: concepts/architecture/adapters      
+      children:           
```

Comment:
Do you mind removing the other link and leaving this one?

Reply:
Sure , No problem . 
Actually I did it because of the easy discoverability  of the adapters section . I did knew that its the part of architecture .